style/pages/projects-details.css

```diff
@@ -1,11 +1,14 @@
 .projects-details .grid {
   display: grid;
-  grid-template-columns: 380px 1fr 320px;
+  grid-template-columns: 350px 1fr 300px;
 }
 
 .projects-details .related-projects-list {
   background-color: var(--color-bg);
   border-right: 1px solid var(--color-border);
+  position: sticky;
+  top: 66px;
```

Comment:
I suppose this is `app-header` height. If you're starting to use some other components' dimensions somewhere outside, it's probably the time to introduce a variable, which will hold that value.